app/static/joke.js

```diff
@@ -0,0 +1,14 @@
+function makejoke() {
+  fetch('/joke')
+    .then(response => response.json())
+    .then(data => alert(data.text));
+}
+
+
+function addEventsAfterPageLoaded() {
+  const element = document.getElementById("ajoke");
+  element.addEventListener("click", makejoke, false);
+  };
+
+
+document.addEventListener("DOMContentLoaded", addEventsAfterPageLoaded);
```

Comment:
Awesome!

Reply:
I received help for that one.
Thanks :)